src/Nfts/ImageGallery/ImageGallery.tsx

```diff
@@ -58,81 +91,133 @@ interface ModeratedImageProps {
 
 const ModeratedImage = ({fingerprint, image, text, onPress}: ModeratedImageProps) => {
   const wallet = useSelectedWallet()
-  const moderationStatusQuery = useQuery({
-    queryKey: [wallet.id, 'nft', fingerprint],
-    queryFn: () => wallet.fetchNftModerationStatus(fingerprint),
-  })
+  const moderationStatusQuery = useNftModerationStatus({wallet, fingerprint})
```

Comment:
if it is using `suspense` remember to wrap it within a `<Boundary>` component

Reply:
This one is not using suspense, so should be all fine